default.json

```diff
@@ -0,0 +1,25 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    ":combinePatchMinorReleases",
+    ":dependencyDashboard",
+    ":pinAllExceptPeerDependencies",
+    ":separateMajorReleases",
+    "config:base",
+    "group:monorepos",
+    "schedule:weekends",
+    "github>bitwarden/renovate-config:pin-actions"
+  ],
+  "commitMessagePrefix": "[deps]:",
+  "commitMessageTopic": "{{depName}}",
+  "prConcurrentLimit": 0,
+  "lockFileMaintenance": {
+    "enabled": true,
+    "schedule": [
+      "every 2nd week starting on the 2 week of the year before 4am on Monday"
```

Comment:
❓ Was `maintainLockFilesWeekly` too often for you?

Reply:
I've updated the PR description with some context.